ouroboros-network/src/Ouroboros/Network/NodeToNode.hs

```diff
@@ -342,7 +342,7 @@ blockFetchProtocolLimits MiniProtocolParameters { blockFetchPipeliningMax } = Mi
     -- relaxed limit here.
     --
     maximumIngressQueue = addSafetyMargin $ fromIntegral $
-      max (10 * 2_097_154) (blockFetchPipeliningMax * 65535)
+      max 65535 (blockFetchPipeliningMax * 65535)
```

Comment:
Let's keep the ingress queue size:
```suggestion
   maximumIngressQueue = addSafetyMargin $
      max (10 * 2_097_154) (fromIntegral blockFetchPipeliningMax * 65_535)
```

Reply:
Without this change there was a compiler warning about `10 * 2_097_154 > (maxBound ::Word32)`.

Reply:
I think we need to increase it, but we'll do that in a separate PR.

Reply:
btw, the `maximumIngressQueue` is measured in bytes.

Reply:
With my snippet we won't get the worning because it will be computed as an `Int`
```
maximumIngressQueue :: Int
```

Reply:
```hs
λ (10 * 2_097_154 :: Int) > fromIntegral (maxBound @Word32) 
False
```